generators/dart.js

```diff
@@ -1,317 +1,44 @@
 /**
  * @license
- * Copyright 2014 Google LLC
+ * Copyright 2021 Google LLC
```

Comment:
minor: Why 2021?

Reply:
This was because `generators/dart.js` was created in 2014, but `generators/dart/all.js` was created in 2021, and the latter has been renamed to the former's name (while the former is now `generators/dart/dart_generator.js`).